app/views/export/_invitation.html.erb

```diff
@@ -0,0 +1,50 @@
+<tr>
+<td>
+  <%= invitation.invitee.try(:full_name) %> <%= invitation.email %>
+</td>
+<td>
+  <%if invitation.decision.major_version%>
+    v<%= invitation.decision.major_version %>.<%= invitation.decision.minor_version %>
+    <%= invitation.decision.verdict.try(:humanize) %>
+  <% end %>
+</td>
+<td>
+  <%if !invitation.pending? %>
+    <%if invitation.invited? %>
+      Invited <%= invitation.invited_at%>
+    <% else %>
+      <%if invitation.accepted? %>
+        <%if ReviewerReport.for_invitation(invitation) %>
+          <%if ReviewerReport.for_invitation(invitation).status == 'completed' %>
+            Completed <%= ReviewerReport.for_invitation(invitation).datetime %>
+          <% else %>
+            <%if ReviewerReport.for_invitation(invitation).due_datetime.due_at %>
```

Comment:
I ran into
```console
ActionView::Template::Error: undefined method `due_at' for nil:NilClass
/Users/bmiller/PLOS/Aperta/app/views/export/_invitation.html.erb:21:in `_app_views_export__invitation_html_erb___284069628093107782_70157772688800'
/Users/bmiller/.rvm/gems/ruby-2.3.6@tahi/gems/actionview-4.2.11.1/lib/action_view/template.rb:145:in `block in render'
/Users/bmiller/.rvm/gems/ruby-2.3.6@tahi/gems/activesupport-4.2.11.1/lib/active_support/notifications.rb:166:in `instrument'
```
So there might be some nils hidden around there.  Unfortunately I didnt log out the offending manuscripts, but this is what I sampled
```
pbio.2000689.zip	pbio.2001731.zip	pbio.2002346.zip	pbio.2003884.zip	pbio.2004500.zip	pbio.2005141.zip	pbio.2006543.zip
pbio.2000998.zip	pbio.2001936.zip	pbio.2003142.zip	pbio.2003918.zip	pbio.2004545.zip	pbio.2005171.zip	pbio.2006966.zip
pbio.2001189.zip	pbio.2002183.zip	pbio.2003417.zip	pbio.2004218.zip	pbio.2005050.zip	pbio.2006069.zip	pbio.2006976.zip
```
though im not actually sure its one of those

Reply:
Thanks for checking. Where can I get data to test these manuscripts? It's been a while and I just have the seed data.